templates/default/elasticsearch-env.sh.erb

```diff
@@ -0,0 +1,26 @@
+# JVM Configuration for ElasticSearch
```

Comment:
:+1: I'd love to make this simpler; I didn't make any assumptions about why this was needed, so I just brought it over. Let's get rid of it.